__tests__/app.test.js

```diff
@@ -95,4 +95,44 @@ describe('GET /api/articles/', () => {
             });
         });
     });
+
+    describe('/api/articles', () => {
+        test('GET 200: should respond with an articles array of article objects, each of which should have the following properties: author, title, article_id, topic, created_at, votes, article_img_url, comment_count, (total count of all the comments with this article_id)', () => {
```

Comment:
I'd really like you to write your own test descriptions, not copy and paste the instruction from the L2C instructions.